projects/manager/src/app/permissions/permissions.component.html

```diff
@@ -14,10 +14,51 @@
         <td mat-cell *matCellDef="let element">{{ element.permission }}</td>
     </ng-container>
 
+    <ng-container matColumnDef="remove">
+        <th mat-header-cell *matHeaderCellDef>Remove</th>
+        <td mat-cell *matCellDef="let element">
+            <button mat-icon-button (click)="removePermission(element)">
+                <mat-icon>delete</mat-icon>
+            </button>
+        </td>
+    </ng-container>
+
     <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
     <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
 </table>
 
 <mat-paginator [pageSize]="5" [pageSizeOptions]="[5, 10, 20]"></mat-paginator>
 
+<form [formGroup]="form">
+    <div class="flex-container">
+        <mat-form-field>
+            <mat-label>Role Specification</mat-label>
+            <mat-select formControlName="roleSpec">
+                <mat-option class="options" *ngFor="let roleSpec of ROLE_SPECS" [value]="roleSpec">
+                    {{ roleSpec }}
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+        <mat-form-field appearance="fill">
+            @if (form.get('roleSpec')?.value === ROLE_SPEC.Id) {
+                <mat-label>Role Id</mat-label>
+            } @else {
+                <mat-label>Role Name</mat-label>
+            }
+            <input matInput type="text" formControlName="role" />
+        </mat-form-field>
+        <mat-form-field>
+            <mat-label>Permission</mat-label>
+            <mat-select formControlName="permission">
+                <mat-option class="options" *ngFor="let permission of PERMISSIONS" [value]="permission">
+                    {{ permission }}
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+        <button mat-raised-button class="add-permission" color="primary" [disabled]="form.pristine" (click)="addPermission()">
+            Add permission
+        </button>
+    </div>
+</form>
+
```

Comment:
![grafik](https://github.com/geo-engine/geoengine-ui/assets/4947061/5182e1d4-185d-4ce5-b171-a0de656939e8)

There can be two owners, this seems buggy.